src/components/PeopleFilters.tsx

```diff
@@ -27,66 +92,54 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
+            {['16', '17', '18', '19', '20'].map(century => {
```

Comment:
It's bad practice to use arrays like this, put it in a separate variable outside of the component